hw1_collections/hw_collections.py

```diff
@@ -0,0 +1,89 @@
+# List of dictionaries for translating sequences #
+
+complement_dna = {'A':'T', 'T':'A', 'G':'C', 'C':'G',
+                  'a':'t', 't':'a', 'g':'c', 'c':'g'}
+
+complement_rna = {'A':'U', 'U':'A', 'G':'C', 'C':'G',
+                  'a':'u', 'u':'a', 'g':'c', 'c':'g'}
+
+transcribe_dna_to_rna = {'A':'A', 'T':'U', 'G':'G', 'C':'C',
+                         'a':'a', 't':'u', 'g':'g', 'c':'c'}
+
+transcribe_rna_to_dna = {'A':'T', 'U':'A', 'G':'C', 'C':'G',
+                         'a':'t', 'u':'a', 'g':'c', 'c':'g'}
+
+allowed_symbols = ['A', 'T', 'G', 'C', 'U',
+                   'a', 't', 'g', 'c', 'u']
+
+# List of commands #
+
+def complement(sequence):
+    if any(U in sequence for U in ['U', 'u']):
```

Comment:
Плохо читается, лучше так
```suggestion
    if "U" in sequence or "u" in sequence:
```